docs/static/keystore.asciidoc

```diff
@@ -12,23 +12,38 @@ place of the secret value when you configure sensitive settings.
 The syntax for referencing keys is identical to the syntax for
 <<environment-variables, environment variables>>:
 
+[source,txt]
+-----
 `${KEY}`
+-----
 
 Where KEY is the name of the key.
 
-For example, imagine that the keystore contains a key called `ES_PWD` with the
-value `yourelasticsearchpassword`:
+**Example**
+ 
+Imagine that the keystore contains a key called `ES_PWD` with the value `yourelasticsearchpassword`.
 
-* In configuration files, use: `output { elasticsearch {...password => "${ES_PWD}" } } }`
-* In `logstash.yml`, use: `xpack.management.elasticsearch.password: ${ES_PWD}`
+In configuration files, use: 
+
+[source,shell]
+-----
+output { elasticsearch {...password => "${ES_PWD}" } } } 
+-----
+
+In `logstash.yml`, use:
+
+[source,shell]
+-----
+xpack.management.elasticsearch.password: ${ES_PWD}
+-----  
 
 Notice that the Logstash keystore differs from the Elasticsearch keystore.
 Whereas the Elasticsearch keystore lets you store `elasticsearch.yml` values by
 name, the Logstash keystore lets you specify arbitrary names that you
 can reference in the Logstash configuration.
 
-NOTE: There are some configuration fields that have no secret meaning, so not every field could leverage
-the secret store for variables substitution. Plugin's `id` field is a field of this kind
+NOTE: Some configuration fields that have no secret meaning, so not every field can use
```

Comment:
I think there is an extra word in here:

```suggestion
NOTE: Some configuration fields have no secret meaning, so not every field can use
```

Reply:
This NOTE is a little confusing - I'm not sure if `secret meaning` is particularly clear or obvious

Reply:
That section could use some work, so I removed it for now. I will follow up with a new issue.